XenModel/Messages.Designer.cs

```diff
@@ -21874,6 +21874,33 @@ public class Messages {
             }
         }
         
+        /// <summary>
+        ///   Looks up a localized string similar to Data merge for VDI &apos;{0}&apos; of VM &apos;{1}&apos; has completed..

```

Comment:
- This file is autogenerated from Messages.resx. Any changes to the keys and their values should be made in the resx file, because next time something is added in the latter, the compilation will wipe out the manual changes in the Designer.cs.
- Since we still have the `*_TITLE` and `*_DESCRIPTION` keys, I would suggest using them, i.e. add the VDI name in the description only to make it more detailed than the title.

Reply:
I removed those keys as they were now going to be the same. Happy to revert tho and make this change :)

Reply:
Not sure how ive achieved this. That was what I had intended but clearly something went awry.